brand/models/commentary.py

```diff
@@ -237,7 +244,13 @@ def clean(self):
         # Ensure no cycles when saving
         self.compute_inherited_rating(throw_error=True)
 
+    # updating last_viewed
     def save(self, *args, **kwargs):
+        if self.pk:
```

Comment:
last_reviewed should probably only be updated from speed rater. Given that, I don't think it makes sense to put it in the save metnod. (Also, as mentioned in a previous comment, last_reviewed will sometimes need to be updated even when the rating hasn't changed.)